python/mag_calibration.py

```diff
@@ -1,3 +1,7 @@
+#!/usr/bin/python
```

Comment:
how would you feel about leaving mag_calibration in a calibration directory? I can see us having a number of calibration routines and it might be good to organize them. is that a problem for importing?